netmaster/objApi/objapi_test.go

```diff
@@ -1318,7 +1320,79 @@ func TestDynamicGlobalVxlanRange(t *testing.T) {
 }
 
 // TestGlobalSetting tests global REST api
+func TestGlobalSettingFwdMode(t *testing.T) {
+	// set to default values (no-op)
+	checkGlobalSet(t, false, "default", "1-4094", "1-10000", "bridge", "proxy", "172.19.0.0/16")
```

Comment:
Since this returns an error, we should probably be asserting that it's nil or some expected value...

If there's a legit reason for skipping error checks on a call, maybe add a comment above the call

Reply:
ah, i was just copy pasting that part, but I can make a helper checkGlobalSetNotNil(..) to avoid if block every call